tests/js/template/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -130,7 +136,8 @@ var o = {
 };
 
 var x = {
-  y: () => Relay.QL\`
+  y: () =>
+    Relay.QL\`
```

Comment:
I don't think this is desired. These tests are completely unrelated to the issue being fixed, so they shouldn't change.

Reply:
The old one, if you insert a empty line before the `Relay.QL`, it format like this too, do we really want distinguish?

**Prettier 2.0.5**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEA3AhgJwAQA9sC82wAOlNtgJ5LYAUAlIQHzZlkXutQBKcANukoA6AIoAZAAacKARwCucTJWLSK2ACTAAZhAgBfADSqKWgJb8AJkfJq9nKVD0BuEAZAQADjFPQAzslAsTAgAdwAFLAR-FHQ+EMF-NwAjTHQwAGs4GABlDzTTKABzZBhMBTcACxgAWz4AdQrTeF88sDhsqKbTVCbKZHBfRJAC30UYMNTC6vRkLVjRtwArX1wAIVSMrOz0argxArhZ+bglleyCwr44ETkIeCO+BZA8zFHMfqT0JP5XZ8wCmB1UwWGAVZAADgADG4PMFRnVUh5+rC4G9UIc3PI7nAJp5oiB0L4ALRQOBwCzk36YODyUzUiboKYzJBzR4nECjaqmEpldm+C5XG7Yh5PGBfIEgsFIABMblK6FMfAuAGEINVpv1UQBWX5yUYAFS+0VZT1QCgAklAKbBsmB-l4AIJW7IwShXEVwPR6IA)
```sh
--parser babel
```

**Input:**
```jsx
var x = {
  y: () => 

  
  
Relay.QL`
    query {
      ${foo},
      field,
    }
  `
};
```

**Output:**
```jsx
var x = {
  y: () =>
    Relay.QL`
    query {
      ${foo},
      field,
    }
  `,
};

```

Reply:
Unfortunately, yes. I don't like it, but this is intentional. You can use git blame to find the respective PRs and discussion.

Reply:
For reference, that PR was https://github.com/prettier/prettier/pull/1497
I'd still like to find a way to eliminate this dependency on the input formatting.